pages/_document.js

```diff
@@ -104,8 +105,14 @@ export default class MyDocument extends Document {
           <meta name='msapplication-config' content='/static/browserconfig.xml' />
           <meta name='referrer' content='no-referrer' />
         </Head>
-        <body>
+        <body style={{ fontSize: `${DEFAULT_FONT_SIZE}px` }}> /* prevent unwanted resize of non article elements */
           {!nojs && <script dangerouslySetInnerHTML={{ __html: `var _paq = _paq || [];` }} />}
+          {!nojs && <script dangerouslySetInnerHTML={{ __html: `
+            (function() {
+                var fontSize = window.localStorage.getItem('${FONT_SIZE_KEY}')
+                fontSize = fontSize ? fontSize : ${DEFAULT_FONT_SIZE}
+                document.documentElement.style.fontSize = fontSize + 'px'
+              })();` }} />}
```

Comment:
Yes – maybe mine is a little too verbose indeed. I have to admit that I find long one liners a bit hard to decipher at times so I prefer to avoid them.